specification/loadtestservice/LoadTestService/routes.tsp

```diff
@@ -472,7 +472,7 @@ numeric, underscore or hyphen characters.
 
       @doc("The aggregation")
       @query
-      aggregation: string;
+      aggregation?: string;
```

Comment:
I'm just curious if these changes will break the other languages...I dont believe we saw issues with these, although I will admit I think your definitions here are correct according to the swagger. cc @msyyc @chunyu3 @MaryGao could you all also take a look?

If this does change anything for an emitter does it mean that there was a different assumption when the `required` property wasnt included in swagger?

Reply:
The change is expected for python. We don't find it because of the missing function of changelog tool: https://github.com/Azure/azure-sdk-for-python/issues/31219

Reply:
The change is expected for .NET.

Reply:
The change is expected for JavaScript also, we didn't report it because the property optionality changing from required to optional is not a breaking for JS.